src/Dispatcher.php

```diff
@@ -85,7 +85,7 @@ private function processRouteInfo(array $dispatch)
         $controller = $this->container->get($dispatch[1][0]);
         $method = $dispatch[1][1];
         $params = $dispatch[2];
-        $function = (new ReflectionClass($controller))->name;
```

Comment:
I opened the issue just to point out that ReflectionClass is faster than get_class :D